docs/examples/automatically-blur-nsfw-images.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: default
+title: Blur a NSFW image
+nav_order: 13
+parent: Examples
+---
+
+# Introduction
+We're going to be creating a workflow that takes an image via a URL and checks if it is safe for work then outputs the response either blurred or unblurred depending on the contents.
+
+We will need 'imagerecognition' to describe whether the image is safe for work, 'blur' container to blur the image and a simple requester container to get the contents of the url.
```

Comment:
**Suggested rewrite:**

This workflow requires three functions:
1. The ["imagerecognition"](https://hub.docker.com/r/vorteil/imagerecognition) container to determine if the image at a URL is safe for work.
2. The ["blur"](https://hub.docker.com/r/vorteil/blur) container to fetch and apply a blur filter to the image at the URL.
3. The ["request"](https://hub.docker.com/r/vorteil/request) container to fetch the unaltered image at the URL.